content/docs/io/overview.md

```diff
@@ -6,7 +6,119 @@ menu:
     parent: io
 ---
 
-This page has not been worked on yet. If you'd like to contribute visit the [doc-push]
-repo.
+The Rust standard library provides support for networking and I/O, such
+as TCP connections, UDP sockets, reading from and writing to files, etc.
+However, those operations are all synchronous, or _blocking_, meaning
+that when you call them, the current thread may stop executing and go to
+sleep until it is unblocked. For example, `read` will block until there
```

Comment:
nit: should you mention what `read` does? Where does it read information? If I've never heard of `read` before will be I confused by this?